app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationInfo.scss

```diff
@@ -36,10 +38,23 @@
   height: 108px;
   object-fit: contain;
 }
+.header {
+  display: flex;
+  margin-top: 24px;
+}
+
+.buttons-section {
+  display: flex;
+  flex-direction: row-reverse;
+  flex-shrink: 0;
+  margin-left: auto;
+  button {
+    margin-right: 8px;
```

Comment:
Looks like this margin apply for each button but you don't need add margin to last one